install-dev.sh

```diff
@@ -16,7 +16,7 @@ git clone https://git.openwrt.org/project/libubox.git --depth=1
 git clone https://git.openwrt.org/project/uci.git --depth=1
 { cd uci && cmake . && make install && cd ..; } || { echo 'Installing uci failed!' && exit 1; }
 # install nixio
-luarocks install https://raw.githubusercontent.com/Neopallium/nixio/master/nixio-scm-0.rockspec
+luarocks install https://luarocks.org/nixio-0.3-1.src.rock
```

Comment:
The CI is failing due to this line: https://github.com/openwisp/openwrt-openwisp-monitoring/runs/6129392297?check_suite_focus=true#step:6:674

I tried by pointing url to luarocks.org but it shows error in building: https://github.com/openwisp/openwrt-openwisp-monitoring/runs/6131278631?check_suite_focus=true#step:6:690

@nemesisdesign, @devkapilbansal Do you know how to resolve this error?

Reply:
Why you changed the url?

Reply:
Because the command was failing with previous url, see :point_down: 

![image](https://user-images.githubusercontent.com/56037184/164878942-2b2f19ea-70f6-4e81-910a-0e9d238634ef.png)

Command:
```
(base) ➜  openwrt-openwisp-monitoring git:(issues/84-agent-exits) sudo luarocks install https://raw.githubusercontent.com/Neopallium/nixio/master/nixio-scm-0.rockspec
Cloning into 'nixio'...
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.

Error: Failed cloning git repository.
```